src/main/java/Main2.java

```diff
@@ -51,12 +66,25 @@ public static void main(String[] args) {
         UpdateController updateController = new UpdateController(updateUseCase);
         JPanel screen4 = new UpdateScreen(allScreens, updateController);
         // similar to above
+        // screen4 is all about "update price" function.
+        ImportOutputBoundary ImportPresenter = new ImportPresenter();
+        ImportInputBoundary ImportUseCase = new Import(ImportPresenter);
+        ImportController ImportController = new ImportController(ImportUseCase);
+        JPanel screen5 = new ImportScreen(allScreens, ImportController);
+
+        ExportOutputBoundary ExportPresenter = new ExportPresenter();
+        ExportInputBoundary ExportUseCase = new Export(ExportPresenter);
+        ExportController ExportController = new ExportController(ExportUseCase);
+        JPanel screen6 = new ExportScreen(allScreens, ExportController);
```

Comment:
These changes look good. I'll probably incorporate this into a director/builder framework.

Reply:
After you resolve the conflicting files, I think it's ready to be merged.